README.md

```diff
@@ -34,7 +35,50 @@ var User = Arrow.Model.extend('user', {
 			attachment_content: { name: 'content', type: String }
 		},
 		connector: 'appc.mongo'
-	});
+	})
+```
+
+#### Selecting Fields
+
+To select precise fields from a data source, for the majority of cases making a model with fields with keys which match those of a specified model, and specifying the correct type is enough:
+
+```javascript
+Arrow.Model.extend('simpleUser', {
+	fields: {
+			first_name: { type: String, model: 'user' },
+			last_name: { type: String, model: 'user' }
+	},
+	connector: 'appc.composite'
+})
+```
+
+In some cases this isn't precise enough. Using a field of type: Array or Object can result in "Selecting Whole Models Instead of Fields" or "Joining with Multiple Children" as described below. To make sure that the response is always as expected, when selecting individual fields from models, always specify a "name" property, even when it's identical to the field key.
+
+```javascript
+Arrow.Model.extend('simpleUser', {
+	fields: {
+			first_name: { type: String, model: 'user', name: 'first_name' }
+			last_name: { type: String, model: 'user', name: 'last_name' },
+			roles: {type: Array, model: 'user', name: 'roles'}
+	},
+	connector: 'appc.composite'
+})
+```
+
+#### Renaming/Aliasing Fields
+
+The composite connector also allows fields to be named differently from their undelying data source.
+In this example, we are accessing the fields first_name and last_name originating in the user model, as author_first_name and author_last_name in the new model. 
```

Comment:
Maybe i'll just remove that first paragraph you commented on and just document that you *must* use "name" for selecting specific fields no matter what the type.